spec/middleware-1.0.0-draft.1.md

```diff
@@ -0,0 +1,309 @@
+# OWIN Middlewares
+
+**Version**
+1.0.0-draft.1
+
+**Authors**
+Sebastien Lambla (Caffeine IT)
+
+**License**
+[Creative Commons Attribution 3.0 Unported License][CCLicense]
+
+**Last updated**
+1 October 2015
+
+## Abstract
+
+The [OWIN Specification][owin-spec] defines a standard interface between .NET web servers
+and web applications. This document specifies additional standard interfaces
+for reusable OWIN Middleware components.
+
+## Status
+
+This is a working draft.
+
+This document is a submission to the OWIN Working Group and is valid for a
+period of six months and may be updated, replaced, or obsoleted by other
+documents at any time. It is inappropriate to use working drafts as reference
+material or to cite them other than as "work in progress"
+
+This working draft will expire on 1 March 2016.
+
+## Copyright
+
+Copyright (c) 2014 the persons identified as the document authors. The work
```

Comment:
Standard RFC notice. Copyright is exclusively by authors and editors by definition of what copyright is. The previous assignment of copyrigh tto "the owin working group" would assume it has legal existence, which under most jurisdiction it doesn't.